test/object-store/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ if(REALM_ENABLE_SYNC)
         sync/sync_test_utils.cpp
         sync/user.cpp
         sync/app.cpp
+        sync/test_ez_websocket.cpp
```

Comment:
the convention here is that we don't need the "test" prefix on this file name because we can infer it from the directory structure "test/object-store/..."